Feat/quickstart

hrzn opened the pull request:
Solves #734 

A brand new quickstart notebook :)

brunnedu reviewed (approved):
Looks good! Thanks a lot for your effort in going through this huge notebook, I think it will be very helpful for people starting out with Darts.

I made a few comments in ReviewNB.

hrzn commented:
> Looks good! Thanks a lot for your effort in going through this huge notebook, I think it will be very helpful for people starting out with Darts.
> 
> I made a few comments in ReviewNB.

Thank you @brunnedu ! I've addressed your comments.
